src/main/kotlin/no/nav/helse/fritakagp/processing/gravid/krav/GravidKravProcessor.kt

```diff
@@ -179,12 +171,15 @@ class GravidKravProcessor(
                     dokumentVarianter = listOf(
                         DokumentVariant(
                             fysiskDokument = it.base64Data,
-                            filtype = if (it.extension == "jpg") "JPEG" else it.extension.uppercase()
+                            filtype = if (it.extension == "jpg") "JPEG" else it.extension.uppercase(),
+                            variantFormat = "ARKIV",
```

Comment:
At filer som ikke er pdf blir markert med variantFormat Arkiv tror jeg er feil.
https://confluence.adeo.no/display/BOA/opprettJournalpost#:~:text=Typen%20variant%20som,for%C2%A0automatisk%20saksbehandling

Reply:
La oss se på denne på mandag. Mente at det uansett kun er mulig å laste opp PDF, så trenger vi dette? 
Var nylig en stygg bug CVE-2023-4863 rundt opplasting av bilder (det gjaldt webp, men likevel) - best å begrense til PDF uansett.